### DISCUSSION TITLE
Have inverted modes syntax highlighting available in all modes.

### DISCUSSION THREAD

**Discussion by p-stevenson** [johnfactotum/foliate#1393] [Ideas]
Hey not sure if this is just a side effect of how you've implented the inverted dark modes but, having syntax highlighting for code snippets is really nice. Would be great if this could be an option across all themes. Any chance this might be possible?

![Foliate_inverted_syntax_highlighting](https://github.com/user-attachments/assets/6aa3ec5f-c6fb-4d1c-995b-92ac273e3d6a)

**Comment by johnfactotum** [comment]
It isn't related to inverting. The invert mode simply takes the light mode and invert it. The problem is that it forces colors even in light mode for all non-default themes. See #1315.
